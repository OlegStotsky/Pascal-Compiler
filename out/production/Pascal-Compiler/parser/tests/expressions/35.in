var
	a : char;

begin
	a := a + 1 + 2 + a;
end;