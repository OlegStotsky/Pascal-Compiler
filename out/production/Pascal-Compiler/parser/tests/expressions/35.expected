Error at line 4, column 21 : Illegal operand types of operation +
